packages/shared/src/utils/url/helpers.js

```diff
@@ -23,7 +21,6 @@ export const getPlatformFromUrl = (domain = window.location.hostname) => {
     const resolutions = {
         is_staging_deriv_crypto: /^staging-app\.derivcrypto\.com$/i.test(domain),
         is_staging_deriv_app: /^staging-app\.deriv\.com$/i.test(domain),
-        is_deriv_crypto: /^app\.derivcrypto\.com$/i.test(domain),
         is_deriv_app: /^app\.deriv\.com$/i.test(domain),
         is_test_link: /^(.*)\.binary\.sx$/i.test(domain),
```

Comment:
Can we rename these values and update with the proper ones too?

Reply:
OK, sure